### ISSUE TITLE
NER different results for list of single document and list of many documents

### ISSUE THREAD

**Issue by twelveth** [deepmipt/DeepPavlov#951]
I use pretrained model for NER:
```
ru_ner_model = build_model(configs.ner.ner_rus, download=True)
```
 and I get different results, depending on whether I give single document or batch of documents:
![image](https://user-images.githubusercontent.com/27743016/62049026-06d31700-b217-11e9-9830-a782df6ef3ae.png)

**Comment by vikmary** [comment]
The issue if valid only for `configs.ner.ner_rus`, logits for `configs.ner.ner_ontonotes_mult` and `configs.ner.ner_dstc2` do not depend on samples from batch.

```python
from deeppavlov import configs, build_model

ner_model = build_model(configs.ner.ner_conll2003_bert, download=True)

ner_model(['Мото Гран-при Каталонии 2005'])
ner_model(['0'*10, 'Мото Гран-при Каталонии 2005'])

ner_model = build_model(configs.ner.ner_dstc2, download=True)

ner_model(['hi, how are you?'])
ner_model(['0'*10, 'hi, how are you?'])
```

**Comment by K-Mike** [comment]
Maybe try 
`with torch.no_grad():
       ...
`

**Comment by yoptar** [comment]
I'll close this issue as it will most likely never be fixed.
Newer Bert-based models do not have this problem.
